configs/llm-360-amber-s3.yaml

```diff
@@ -0,0 +1,508 @@
+run_name: llm-360-amber-data-hyper-init-repro-3
```

Comment:
Just to be consistent, do we want to call this amberish7, or amberish7-base, something like that (I think this name predates the amberish moniker)?